src/sentry/replays/query.py

```diff
@@ -221,7 +223,9 @@ def query_replays_dataset_tagkey_values(
         aggregated_column = Function("identity", parameters=[grouped_column], alias="tag_value")
 
     if tag_like:
-        where.append(Condition(grouped_column, Op.LIKE, tag_like))
+        where.append(
+            Condition(Function("lower", parameters=[grouped_column]), Op.LIKE, tag_like.lower())
```

Comment:
Just putting this here as a FYI, wild card searches on strings for other datasets previously used ClickHouse's [match](https://clickhouse.com/docs/en/sql-reference/functions/string-search-functions#match) function but was timing out on the spans dataset. One workaround I found was to use [positionCaseInsensitive](https://clickhouse.com/docs/en/sql-reference/functions/string-search-functions#positioncaseinsensitive) for these types of queries which performed a lot better. (I also have special cases for startsWith and endsWith).

I haven't tested the performance of `LIKE` but just wanted to share this in case you find it useful.

Reply:
Thanks! Did you implement it with something like
`Condition(Function("positionCaseInsensitive", parameters=[]), Op.NEQ, 0)`?